configs/foreman/nightly.yaml

```diff
@@ -361,56 +359,6 @@
       foreman-plugins-nightly-rhel7-dist:
         foreman-plugins-nightly-rhel7: 2
         foreman-plugins-nightly-nonscl-rhel7: 0
-  - name: foreman-client-nightly-rhel6
```

Comment:
This would probably also need to be reflected in foreman-packaging so it won't attempt to build anymore.